src/functions/template-tags/date.php

```diff
@@ -148,3 +148,172 @@ function tribe_get_datetime_separator( $default = ' @ ', $esc = false ) {
 		return apply_filters( 'tribe_datetime_separator', $separator );
 	}
 }//end if
+
+if ( ! function_exists( 'tribe_get_start_time' ) ) {
+	/**
+	 * Start Time
+	 *
+	 * Returns the event start time
+	 *
+	 * @category Events
+	 * @param int    $event       (optional)
+	 * @param string $dateFormat  Allows date and time formating using standard php syntax (http://php.net/manual/en/function.date.php)
+	 * @param string $timezone    Timezone in which to present the date/time (or default behaviour if not set)
+	 *
+	 * @return string|null Time
+	 */
+	function tribe_get_start_time( $event = null, $dateFormat = '', $timezone = null ) {
```

Comment:
I realize this is existing code and you're just relocating it ... but here and elsewhere we could use this opportunity to switch to snake_cased args (totally optional).